pkg/solana/txm/txm.go

```diff
@@ -782,9 +782,9 @@ func (txm *Txm) GetTransactionStatus(ctx context.Context, transactionID string)
 	}
 
 	switch state {
-	case txmutils.Broadcasted:
+	case txmutils.Broadcasted, txmutils.Processed:
 		return commontypes.Pending, nil
-	case txmutils.Processed, txmutils.Confirmed:
+	case txmutils.Confirmed:
 		return commontypes.Unconfirmed, nil
```

Comment:
Maybe rename `commontypes.Unconfirmed` as well? `pending -> confirmed -> finalized`

Reply:
I agree that this naming is confusing... I believe it stems from a misalignment on what the ChainReader defines as Unconfirmed and how the TXM defines it. I think aligning those changes is out of scope for this PR, but I've created [a ticket](https://smartcontract-it.atlassian.net/jira/software/c/projects/BCFR/boards/358?assignee=61a6f090ef18ca00717f84db&selectedIssue=BCFR-1111) for it to address in the future.